pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/util/Auth0Util.java

```diff
@@ -337,6 +337,41 @@ public Map<String, String> getAuth0UsersByEmails(Set<String> emailIds, String mg
         return results;
     }
 
+    public Map<String, String> getAuth0UsersByAuth0UserIds(Set<String> userIds, String mgmtApiToken) {
```

Comment:
Can we add a test for this?  Maybe call `createAuth0User()` and verify that the right emails come back from `getAuth0UsersByAuth0UserIds()`.

Reply:
I will first look for any existing tests which create new Auth0 users and add a test there to check this method.
Want to avoid creating new real users unless I really have to. If I can't find any, will create one.